Cabal/src/Distribution/Fields/ParseResult.hs

```diff
@@ -188,7 +188,14 @@ parseString
     -> IO a
 parseString parser verbosity name bs = do
     let (warnings, result) = runParseResult (parser bs)
-    traverse_ (warn verbosity . showPWarning name) warnings
+
+    let noWarn = [wt | wt <- [PWTOther ..]
+            , (wt == PWTExperimental) && isVerboseNoWarnExperimental verbosity
+            ]
```

Comment:
If we ever want to add more warning suppressants for cabal parsing warnings, this fella here helps a tiny little bit with that, we just need to pop them in there.

Reply:
BTW, what is `PWTOther`? Is it `minBound` by any chance?

Reply:
I just picked the first on the list, but you have a good point, I should have used `minBound`, because the type does have the instances.

Reply:
The advantage of `minBound` is that it's clear it's the whole list. The disadvantage is that it's not clear what the type of the list is. But here it's obvious from the context.